plugins/aws/s3/bucketEncryptionInTransit.js

```diff
@@ -86,5 +97,98 @@ module.exports = {
             }
         }
         callback(null, results, source);
+    },
+    remediate: function(config, cache, settings, resource, callback) {
+        var putCall = this.actions.remediate;
+        var pluginName = 'bucketEncryptionInTransit';
+        var bucketNameArr = resource.split(':');
+        var bucketName = bucketNameArr[bucketNameArr.length - 1];
+
+        // find the location of the bucket needing to be remediated
+        var bucketLocations = cache['s3']['getBucketPolicy'];
```

Comment:
change the variable name to reflect the data

```suggestion
        var bucketPolicy = cache['s3']['getBucketPolicy'];
```

Reply:
changed